assets/scripts/storage.js

```diff
@@ -0,0 +1,53 @@
+// Saves new goals
```

Comment:
Nit: Could you follow the function comment headers that the main files uses?

(sorry..)

Reply:
Resolved